moto/events/utils.py

```diff
@@ -1,3 +1,25 @@
+from typing import TYPE_CHECKING, Any, Dict, List, TypedDict, Union
```

Comment:
The `Any` and `Dict` should probably also be only imported when `TYPE_CHECKING` - the linter is currently failing:
https://github.com/getmoto/moto/actions/runs/7594740349/job/20686572806?pr=7234